rawfile/declarative.py

```diff
@@ -40,6 +41,14 @@ def be_mounted(dev, mountpoint):
             return
         # noinspection PyUnreachableCode
         be_unmounted(mountpoint)
+    elif mountpoint.resolve().is_file():
+        opts = ["bind"]
+        # don't we need to add this?
+        # if readonly:
```

Comment:
the mount mode also doesn't set ro, is this somehow handled by the kubelet or is this a bug?

Reply:
You mean that when we mount a block volume in readonly mode we will not receive that the volume mount is readonly?

Reply:
I just tested this, I set the readOnly of the PVC in a pod, and I see the readonly being set to true, but the device is still writeable...
Also though this commented out code here doesn't work, seems that the readonly flag is only in effect for the rawfile pod, and even then it's still writeable :)
Btw, added an enum for the access type, ptal

Reply:
Looks like we can't set the readonly flag on bind mounts for block devices, need to change the block device itself.
We can do this but... this wouldn't work when multiple pods have readonly flag set to a mix of true and false

Reply:
OK, I See, We should mention that we are not supporting readonly mounts